starknet_py/tests/e2e/fixtures/misc.py

```diff
@@ -94,6 +99,47 @@ def get_tx_status_full_node_client():
     return f"{FullNodeClient.__module__}.FullNodeClient.get_transaction_status"
 
 
+class ContractVersion(Enum):
+    V1 = "V1"
+    V2 = "V2"
+
+
+class UnkonwnArtifacts(BaseException):
+    pass
+
+
+def load_contract(contract_name: str, version: Optional[ContractVersion] = None):
+    if version is None:
+        if "--contract_dir=v1" in sys.argv:
+            version = ContractVersion.V1
+        if "--contract_dir=v2" in sys.argv:
+            version = ContractVersion.V2
+
+    artifacts_path = CONTRACTS_V2_ARTIFACTS
+    directory = CONTRACTS_V2_COMPILED
+
+    if version == ContractVersion.V1:
+        artifacts_path = CONTRACTS_V1_ARTIFACTS
+        directory = CONTRACTS_V1_COMPILED
```

Comment:
lets use if..else if..else to handle this logic. we will be more explicit, and avoid overwriting variables